cmd/cloudx/client/project.go

```diff
@@ -267,6 +267,9 @@ func (h *CommandHelper) UpdateProject(ctx context.Context, id string, name strin
 	if _, found := interim["name"]; !found {
 		interim["name"] = ""
 	}
+	if _, found := interim["organizations"]; !found {
```

Comment:
Should we maybe have some more generic way here that doesn't require updates every time we add a key? What options do we have for that?

Reply:
yea that'd be good. but the error comes straight from the std lib decoder (I think). Not sure how we can tweak that..